mongo-connector/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM confluentinc/cp-kafka-connect:6.1.9
+
+COPY connect_start.sh /scripts/connect_start.sh
+
+RUN confluent-hub install --no-prompt mongodb/kafka-connect-mongodb:1.8.0
```

Comment:
Does this connect image support only certain Mongo versions? Can we add a comment about that in here if so?